tests/vm_userdata_test.go

```diff
@@ -42,9 +46,16 @@ var _ = Describe("CloudInit UserData", func() {
 	virtClient, err := kubecli.GetKubevirtClient()
 	tests.PanicOnError(err)
 
-	BeforeEach(func() {
-		tests.BeforeTestCleanup()
-	})
+	Dial := func(vm string, console string) *websocket.Conn {
+		wsUrl, err := url.Parse(flag.Lookup("master").Value.String())
+		Expect(err).ToNot(HaveOccurred())
+		wsUrl.Scheme = "ws"
+		wsUrl.Path = "/apis/kubevirt.io/v1alpha1/namespaces/" + tests.NamespaceTestDefault + "/vms/" + vm + "/console"
```

Comment:
Ca you use

```go
url, err := kubecli.NewVirtHandlerClient(virtClient).ForNode("master").ConsoleURI(vm)
url.Scheme = "ws"
```
to construct the URL, still not optimal, but at least we don't distribute the URL everywhere in our code. Also in console_test.go where this was copied from.

Reply:
yep, i'll take a look at that.

Reply:
What you suggest isn't equivalent to what's currently being done here.

ConsoleURI(vm) returns a url directly to virt-handler.  What we're doing in the console tests and this userdata test is contacting the api server.

Reply:
True.